### PR TITLE
fixes #2

### LINKED ISSUE #2: Top Month Always Empty (Archive Sidebar)
- When I click on the top-most month in the Archive Sidebar it says "No posts found...".
- When I create a post in the _current month_ (e.g. February) it displays a link in the sidebar for _next month_ (e.g. March), and says "No posts found..."

### PR DESCRIPTION
**Pull request by alexpsu**
fixes #2 , the month issues work swimingly.

### CODE REVIEW

File: app/assets/stylesheets/sidebar_admin.css.scss [hunk 1]
```diff
@@ -34,3 +34,7 @@
     display: inline-block;
   }
 }
+
+h3.sidebar_title {
```
Comment (justincastilla) [inline app/assets/stylesheets/sidebar_admin.css.scss]: Defend your decision to implement a h3 tag style addition to fix issue #2

File: lib/amazon_sidebar/app/views/amazon_sidebar/_content.html.erb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <% unless sidebar.asins.empty? %>
-  <h3 class="sidebar-title"><%= h sidebar.title %></h3>
+  <h3 class="sidebar_title"><%= h sidebar.title %></h3>
```
Comment (justincastilla) [inline lib/amazon_sidebar/app/views/amazon_sidebar/_content.html.erb]: Defend your decision to change this class assignment to fix issue #2

File: lib/archives_sidebar/lib/archives_sidebar.rb [hunk 3]
```diff
@@ -30,11 +30,11 @@ def parse_request(_contents, _params)
     article_counts = Content.find_by_sql(["select count(*) as count, #{date_func} from contents where type='Article' and published = ? and published_at < ? group by year,month order by year desc,month desc limit ? ", true, Time.now, count.to_i])
 
     @archives = article_counts.map do |entry|
-      month = (entry.month.to_i%12)+1
+      month = (entry.month.to_i%12)
       year = entry.year.to_i
       {
         name: I18n.l(Date.new(year, month), format: '%B %Y'),
-        month: month - 1,
+        month: month-1,
```
Comment (tgaff) [inline lib/archives_sidebar/lib/archives_sidebar.rb:37]: The ruby style guide recommends " Use spaces around operators, after commas, colons and semicolons, around { and before }. Whitespace might be (mostly) irrelevant to the Ruby interpreter, but its proper use is the key to writing easily readable code. [link](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
"

I think this is likely going to fail our rubocop checks.  Please run them and verify.

File: lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <% unless sidebar.archives.blank? %>
-  <h3 class="sidebar_title"><%= sidebar.title %></h3>
-  <div class="sidebar_body">
-    <ul id="archives">
+  <h3 class="sidebar-title"><%= sidebar.title %></h3>
+  <div class="sidebar-body">
+    <ul>
```
Comment (justincastilla) [inline lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb]: Defend your decision to remove the ul id attribute  to fix issue #2

File: app/assets/stylesheets/sidebar_admin.css.scss [hunk 5]
```diff
@@ -34,3 +34,7 @@
...
+  font-family: monospace;
```
Comment (tgaff) [inline app/assets/stylesheets/sidebar_admin.css.scss:39]: Hi, I'm wondering what this was done for.  Is this not something we already have?  Please explain.

File: lib/archives_sidebar/lib/archives_sidebar.rb (hunk 3 above)
Comment (justincastilla) [inline lib/archives_sidebar/lib/archives_sidebar.rb]: Defend your decision to alter the global style guide in mathematics in order to solve issue #2

File: lib/archives_sidebar/lib/archives_sidebar.rb [hunk 6]
```diff
@@ -30,11 +30,11 @@ def parse_request(_contents, _params)
     article_counts = Content.find_by_sql(["select count(*) as count, #{date_func} from contents where type='Article' and published = ? and published_at < ? group by year,month order by year desc,month desc limit ? ", true, Time.now, count.to_i])
 
     @archives = article_counts.map do |entry|
-      month = (entry.month.to_i%12)+1
+      month = (entry.month.to_i%12)
```
Comment (justincastilla) [inline lib/archives_sidebar/lib/archives_sidebar.rb]: What does the deletion of the '+1' achieve go sold issue #2? Please describe your solution.

File: app/assets/stylesheets/sidebar_admin.css.scss (hunk 1 above)
Comment (alexpsu) [inline app/assets/stylesheets/sidebar_admin.css.scss:38]: It fixed an issue from #1. I lost the invisibility cloak so I can't hide this change.

File: app/assets/stylesheets/sidebar_admin.css.scss (hunk 5 above)
Comment (alexpsu) [inline app/assets/stylesheets/sidebar_admin.css.scss:39]: Got rid of it.

File: lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb (hunk 4 above)
Comment (alexpsu) [inline lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb:4]: It got added back.

**Pull request closed without merging**